back/config/locales/kl-GL.yml

```diff
@@ -68,7 +68,7 @@ kl:
     infopage_title: Qupperneq paasissutissiivik
```

Comment:
I can’t see the change to this file, even though there’s a diff now.

Reply:
It's an invisible character here `kan jeg deltage? </h2><p `

Reply:
If you copy both texts to your VSCode you'll see it.

Reply:
I should change it right, so we get no diff?

Reply:
I don't think it matters. The new version is even better as it has a simple space character.

Reply:
But we are trying to match the old source files.

Reply:
Good point.